pkg/printers/internalversion/describe.go

```diff
@@ -960,6 +963,19 @@ func printStorageOSPersistentVolumeSource(storageos *api.StorageOSPersistentVolu
 		storageos.VolumeName, storageos.VolumeNamespace, storageos.FSType, storageos.ReadOnly)
 }
 
+func printFCVolumeSource(fc *api.FCVolumeSource, w PrefixWriter) {
+	lun := "<none>"
+	if fc.Lun != nil {
+		lun = strconv.Itoa(int(*fc.Lun))
```

Comment:
not sure which conversion do you mean here...

`fc.Lun` is `int32` and `Itoa` needs `int`. And all this code is there to protect the printer from `fc.Lun==nil`. Simple `w.Write(*fc.Lun)` looks too dangerous to me.